.github/workflows/triage-issues.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       - name: Mark/Close Stale Issues and Pull Requests
         uses: actions/stale@v3
         with:
-          repo-token: ${{ secrets.HOMEBREW_GITHUB_API_TOKEN }}
+          repo-token: ${{ secrets.HOMEBREW_GITHUB_PUBLIC_REPO_TOKEN }}
```

Comment:
> `# This file is synced from the `.github` repository, do not modify it directly.`

Reply:
Oops, addressing in https://github.com/Homebrew/.github/pull/14